### DISCUSSION TITLE
A82BD513-2C70-0F57-E99A-7C44E33E8AE1

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#91] [General]
# A82BD513-2C70-0F57-E99A-7C44E33E8AE1

This addon adds 404 contact binary stars, also known as W Ursae Majoris variables, to Celestia.

https://celestia.mobi/addon?item=A82BD513-2C70-0F57-E99A-7C44E33E8AE1&lang=en

**Comment by exocritic5457** [comment]
44 Boötis B without the addon is the barycenter for 2 stars. With the addon it becomes a contact binary as it should, yet the Ba and Bb stars aren't removed with the addon installed, causing 4 stars in the same spot. Also why is there no 1SWASP J093010.78+533859.5 anywhere? I would've thought this addon might have had it

**Reply by SevenSpheres** [reply]
Yep, conflict between this addon and the [Binary Star Catalog](https://celestia.mobi/resources/item?item=2825D1DF-C124-1E3B-C4B2-76B5B12D2699)...
